lib/cisco_node_utils/cmd_ref/acl_v4.yaml

```diff
@@ -0,0 +1,25 @@
+# Command Reference Common ACL
+#
+# For documentation please see:
+#   - README_YAML.md
+#
+---
+_template:
+  config_get: 'show run aclmgr'
+  config_get_token: '/^ip access-list <acl_name>$/'
+  config_set: 'ip access-list <acl_name>'
+
+ace:
+  config_get_token_append: '/\s+(<seqno>)\s+(permit|deny)\s+(sctp|tcp|ip|udp|pim|icmp|igmp|gre|nos|pcp|ospf|eigrp|ahp|esp|\d+)\s+(any|host \S+|\S+\/\d+|\S+ [:\.0-9a-fA-F]+)\s+(eq \d+|neq \d+|lt \d+|gt \d+|range \d+ \d+)*\s*(any|host \S+|\S+\/\d+|\S+ [:\.0-9a-fA-F]+)\s*(eq \d+|neq \d+|lt \d+|gt \d+|range \d+ \d+)*( [a-zA-Z0-9\/ ]*)*$'
```

Comment:
Some notes about this regular expression.
1) You don't need the \s+ before the (<seqno>).  You can anchor it... `'/^(seqno)...$/`
2) Instead of `(permit|deny)` `(sctp|tcp|ip|udp|pim...)` etc, just return any non-whitespace value `(\S+)`  That way if new values are supported in the future at that position we already handle it.
3) You omitted the trailing `/` to complete the regular expression.